stablehlo/reference/Ops.h

```diff
@@ -206,10 +206,9 @@ Tensor evalXorOp(const Tensor &lhs, const Tensor &rhs, ShapedType resultType);
 /// used for evaluating ops which are not supported by the interpreter.
 /// Assumes that `region` has only one block.
 SmallVector<InterpreterValue> eval(
-    Region &region, ArrayRef<InterpreterValue> args, Process *process = nullptr,
-    Scope *parent = nullptr,
-    function_ref<llvm::Error(Operation &, Process *, Scope &)> fallback =
-        nullptr);
+    Region &region, ArrayRef<InterpreterValue> args,
+    const InterpreterConfiguration *config = nullptr,
```

Comment:
Per earlier comment, I don't think this should be optional here. Let's flip to reference and pass the context through.